recipes/natsort/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "natsort" %}
+{% set version = "4.0.4" %}
```

Comment:
I see that `natsort` is available on the `defaults` channel at `5.0.1` and also `4.0.3`. To follow up on my previous comment, is there a reason to not have conda-forge reflect the newest release?